packages/components/scripts/post-build/react.ts

```diff
@@ -29,6 +29,28 @@ const overwriteEvents = (tmp: boolean) => {
 	writeFileSync(modelFilePath, modelFileContent);
 };
 
+// All things from foundations should be set on the root component - custom "data-" attributes not
```

Comment:
@nmerget I'm not totally sure what you mean by this comment

```suggestion
// All things from foundations should be set on the root component - custom "data-" attributes shouldn't
```